app/models/manageiq/providers/google/network_manager/refresh_parser.rb

```diff
@@ -317,7 +317,7 @@ def get_load_balancer_health_checks
       end
 
       def parse_load_balancer(forwarding_rule)
-        uid = forwarding_rule.id
+        uid = forwarding_rule.id.to_s
```

Comment:
Is the id coming back as an integer type now?  Just making sure this isn't because it was nil...don't want a uid of "" :smile:

Reply:
Well, it's even stranger, since from some endpoints it comes as integer, from some as a string. So doing any resolution in `@data_index` was really painful and messed any consecutive refresh. This one was really fun to debug. :smile:

Reply:
:scream: